src/Sylius/Bundle/CoreBundle/EventListener/OrderShippingListener.php

```diff
@@ -21,24 +21,24 @@
 /**
  * @author Paweł Jędrzejewski <pawel@sylius.org>
  */
-class OrderShippingListener
+final class OrderShippingListener
```

Comment:
You're right, I'm fixing the specs now and removing this class was the best way to fix the related spec :D